09-collaboration.Rmd

```diff
@@ -108,9 +108,9 @@ Package names should be surrounded with speech marks.
 
 ```{r messages=FALSE}
 # Good
-library("ggplot2")
+library("dplyr")
 # Bad
-library(ggplot2)
+library(dplyr)
```

Comment:
I still don't think this is actually 'bad' as it's the fastest way to write package names, not least due to RStudio's autocompletion. How about:

```r
# Non-standard evaluation
library(dplyr)
```